ibm_catalog.json

```diff
@@ -729,21 +885,23 @@
             },
             {
               "key": "secrets_manager_service_plan",
-              "required": true,
+              "type_metadata": "string",
               "virtual": true,
+              "required": true,
               "type": "string",
+              "default_value": "standard",
               "options": [
                 {
+                  "description": "Centrally manage secrets in a dedicated instance.",
```

Comment:
why are we not using the same text as the service tile? I would include all of this text:

<img width="647" height="106" alt="Image" src="https://github.com/user-attachments/assets/b746c0c7-ccee-4ab6-8a5a-e5b208e6e575" />